src/envs/painting.py

```diff
@@ -432,6 +438,26 @@ def render(self,
             "blue = wet+clean, green = dry+dirty, cyan = dry+clean;\n"
             "yellow border = side grasp, orange border = top grasp",
             fontsize=12)
+        # List of NextTo objects to render
+        # Added this to display what objects we are nextto
+        # during video rendering
+        if isinstance(self, envs.RepeatedNextToPaintingEnv):
```

Comment:
can we refactor this in the following way: implement a function called something like `_render_matplotlib` that does everything up until `utils.fig2data(fig)` and returns `fig`. Then, rather than this `isinstance` check here, we can do:

in painting.py:
```
def render(...):
    fig = self._render_matplotlib(...)
    img = utils.fig2data(fig)
    plt.close()
    return [img]
```

and in repeated_nextto_painting.py:
```
def render(...):
    fig = self._render_matplotlib(...)
    nextto_objs = []
    for obj in state:
        ...
    img = utils.fig2data(fig)
    plt.close()
    return [img]
```

one reason for this change would be so that all of the repeated nextto specific code is centralized in that one file.